### PR TITLE
Added parameters for partial and full sentences suggestions

### PR DESCRIPTION
**Pull request by JahanviNShah**
n_best_partial and n_best_full represents the number of partial and full sentences suggestions to be shown for API. The default value is set to 5 partial sentences and 1 full sentence.

### CODE REVIEW

File: mtsimple/api/views.py [hunk 1]
```diff
@@ -95,12 +69,44 @@ def translate_new(request):
     sentence = request.GET.get('sentence')
     partial_trans = request.GET.get('partial_trans', '')
     n_words = request.GET.get('n_words', '')
+
+    #n_best_partial and n_best_full represents the number of partial and full sentence suggestions 
+    n_best_partial = request.GET.get('n_best_partial', '')
```
Comment (SebastinSanty) [inline mtsimple/api/views.py:74]: this does not seem to work for > 5, can you check once from your side?

File: mtsimple/api/views.py [hunk 2]
```diff
@@ -183,4 +189,4 @@ def translate_new(request):
     
     print("sentence", quotaposto(sentence))
     print(quotaposto("என் <unk> என்னை"))
-    return JsonResponse({'result': quotaposto(sentence), 'attn': sumattn, 'partial': L2, 'ppl': perplexity, 'avg': avg_score})
+    return JsonResponse({'result': quotaposto(sentence), 'attn': sumattn, 'partial': L2, 'ppl': perplexity, 'avg': avg_score})
```
Comment (SebastinSanty) [inline mtsimple/api/views.py:192]: nitpick: just add a newline to remove this diff

**Review by SebastinSanty — CHANGES_REQUESTED** [review]
So you can probably leave the `n_best_full` part for #23 . There you might have to change more than just the API code.

Also, can you add add the new parameters to the API doc? Check Swagger documentation https://swagger.io/

File: mtsimple/api/views.py (hunk 1 above)
Comment (JahanviNShah) [reply mtsimple/api/views.py:74]: Yes, it does not work for me too. I am not sure, but seems like somewhere in the model it is hard coded to be of the size 5

**Pull request merged**